R/setup.R

```diff
@@ -45,8 +44,15 @@ source(here::here("R", "helpers.R"))
 # Initialize a dictionary of file paths. See misc/file_dict.csv for details
 paths <- model_file_dict()
 
-# Load the parameters file containing the run settings
-params <- read_yaml("params.yaml")
+# Load the parameters file containing the run settings. The conditional here
+# detects rendering in a Quarto/RMarkdown document and assigns to `model_params`
+# instead, since `params` is a reserved variable name in Quarto
+if (knitr::is_html_output() || knitr::is_latex_output()) {
+  params_obj_name <- "model_params"
+} else {
+  params_obj_name <- "params"
+}
+assign(params_obj_name, read_yaml(here("params.yaml")))
```

Comment:
Quarto reserves the `params` object for the stuff specified in the YAML header. Probably crappy design, but here I use an override to rename the `params` object to `model_params` in a Quarto context. Open to better ideas on how to handle this.